CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ Updates should follow the [Keep a CHANGELOG](http://keepachangelog.com/) princip
 
 ## [Unreleased][unreleased]
 
+### Added
+
+ - The priority of parsers, processors, and renderers can now be set when `add()`ing them; you no longer need to rely on the order in which they are added
+ - Added support for trying multiple parsers per block/inline
+ - Extracted two new bases interfaces from `Environment`:
```

Comment:
`s/bases/base/`

Reply:
I'm slightly embarrassed by how many small issues like this are slipping through...  Thanks for catching this!

Reply:
you may find https://grammarly.com/ browser extension to be useful making minor errors being hilited for you in text areas.